### PR TITLE
doc: prepare go document

### PR DESCRIPTION
**Pull request by SamYuan1990**
Prepare for go document.

Signed-off-by: Sam Yuan <yy19902439@126.com>

### CODE REVIEW

**Comment by SamYuan1990** [comment]
@bestbeforetoday , @channingduan , @davidkhala please free to comments here.

File: go/go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module fabric-admin-sdk
+module github.com/hyperledger-twgc/fabric-admin-sdk
```
Comment (bestbeforetoday) [inline go/go.mod:1]: Does that actually work or will the module path need to be `github.com/hyperledger-twgc/fabric-admin-sdk/go`?

File: go/README.md [hunk 2]
```diff
@@ -1,10 +1,9 @@
...
-## test
-use ginkgo for testing
\ No newline at end of file
+### [Overview](https://github.com/hyperledger/fabric-rfcs/pull/55)
+
+### [Features](https://github.com/Hyperledger-TWGC/fabric-admin-sdk/issues/15)
+
+### [Scenario tests and example](./test/e2e_test.go)
+
+### [Documentation](https://pkg.go.dev/search?q=fabric-admin-sdk&m=package)
```
Comment (bestbeforetoday) [inline go/README.md:9]: Once the module path is correct, this link should look something like:
```
https://pkg.go.dev/github.com/hyperledger-twgc/fabric-admin-sdk
```
or
```
https://pkg.go.dev/github.com/hyperledger-twgc/fabric-admin-sdk/go
```

**Review by bestbeforetoday — COMMENTED** [review]
A couple of queries related to module path

File: go/go.mod (hunk 1 above)
Comment (SamYuan1990) [reply go/go.mod]: well I hope it works, we as don't have go mod at https://github.com/hyperledger/fabric-admin-sdk

File: go/README.md (hunk 2 above)
Comment (SamYuan1990) [reply go/README.md]: change to `https://pkg.go.dev/github.com/hyperledger/fabric-admin-sdk` as ideally

**Comment by SamYuan1990** [comment]
making it to draft to make it same structure with https://github.com/hyperledger/fabric-gateway/blob/main/go.mod#L1

**Comment by SamYuan1990** [comment]
@bestbeforetoday , I just refactored package structure to https://github.com/hyperledger/fabric-gateway please review.

**Review by bestbeforetoday — APPROVED** [review]
This looks good to me. I think it should be possible to have the Go module in a repository sub-directory but version tagging / references become more difficult:

https://go.dev/ref/mod#vcs-version

Having to Go module at the repository root does imply that versioning is consistent across different language implementations within the repository. This is the approach that fabric-gateway takes and it seems to work fine. fabric-gateway intentionally maintains functional parity across all language implementations at specific release versions. It does mean that patch-level version increments that just fix bugs in some language implementations may involve no changes in other languages but this doesn't seem to be a big issue.

**Comment by SamYuan1990** [comment]
merge it as it is.

**Pull request merged**